spec/abstract-operations.html

```diff
@@ -0,0 +1,673 @@
+            <td>
+              String
+            </td>
+            <td>
+              Return _argument_.
+            </td>
+          </tr>
+          <tr>
+            <td>
+              Symbol
+            </td>
+            <td>
+              Return _argument_.
+            </td>
+          </tr>
+          <tr>
+            <td>
+              BigInt
+            </td>
+            <td>
+              Return _argument_.
+            </td>
+          </tr>
+          <tr>
+            <td>
+              <ins>Record</ins>
+            </td>
+            <td>
+              <ins>Return _argument_.</ins>
+            </td>
+          </tr>
+          <tr>
+            <td>
+              <ins>Tuple</ins>
+            </td>
+            <td>
+              <ins>Return _argument_.</ins>
+            </td>
+          </tr>
+          <tr>
+            <td>
+              Object
+            </td>
+            <td>
+              Return _argument_.
+            </td>
+          </tr>
+          </tbody>
+        </table>
+      </emu-table>
+    </emu-clause>
+
+    <emu-clause id="sec-samevaluenonnumeric" aoid="SameValueNonNumeric" oldids="sec-samevaluenonnumber">
+      <h1>SameValueNonNumeric ( _x_, _y_ )</h1>
+      <p>The abstract operation SameValueNonNumeric takes arguments _x_ (an ECMAScript language value) and _y_ (an ECMAScript language value) and returns a completion record whose [[Type]] is ~normal~ and whose [[Value]] is a Boolean. It performs the following steps when called:</p>
+      <emu-alg>
+        1. Assert: Type(_x_) is not Number or BigInt.
+        1. Assert: Type(_x_) is the same as Type(_y_).
+        1. If Type(_x_) is Undefined, return *true*.
+        1. If Type(_x_) is Null, return *true*.
+        1. If Type(_x_) is String, then
+          1. If _x_ and _y_ are exactly the same sequence of code units (same length and same code units at corresponding indices), return *true*; otherwise, return *false*.
+        1. If Type(_x_) is Boolean, then
+          1. If _x_ and _y_ are both *true* or both *false*, return *true*; otherwise, return *false*.
+        1. If Type(_x_) is Symbol, then
+          1. If _x_ and _y_ are both the same Symbol value, return *true*; otherwise, return *false*.
+        1. <ins>If Type(_x_) is Tuple, then</ins>
+          1. <ins>Let _xList_ be _x_'s [[Sequence]] value.</ins>
+          1. <ins>Let _yList_ be _y_'s [[Sequence]] value.</ins>
+          1. <ins>Let _xLen_ be the length of List _xList_.</ins>
+          1. <ins>Let _yLen_ be the length of List _yList_.</ins>
+          1. <ins>If _xLen_ &ne; _yLen_, Return *false*.</ins>
+          1. <ins>Let _k_ be 0.</ins>
+          1. <ins>Repeat, while _k_ &lt; _xLen_,</ins>
+            1. <ins>Let _xItem_ be _xList_[_k_].</ins>
+            1. <ins>Let _yItem_ be _yList_[_k_].</ins>
+            1. <ins>Let _same_ be ! SameValueZero(_xItem_, _yItem_).</ins>
+            1. <ins>If _same_ is *false*, Return *false*.</ins>
```

Comment:
Nit: you could consolidate these two onto the same line, and use complex expressions if you want.